### PR TITLE
adding rapid7 to run tasks partner list

### PR DESCRIPTION
**Pull request by gautambaghel**
### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/integrations/run-tasks/index.mdx [hunk 1]
```diff
@@ -145,6 +145,12 @@ To get started, request early access at [Moderne.io](https://moderne.io/), and f
 
 To get started, follow the instructions in the [Terraform Cloud integration](https://docs.oak9.io/oak9/fundamentals/integrations/setup-terraform-cloud-run-task) user documentation.
 
+### Rapid7
+
+[Rapid7's](https://rapid7.com/) InsightCloudSec enables security teams to provide a single, consistent definition of good across the full CI/CD pipeline, allowing them to evaluate potential risks of Infrastructure as Code (IaC) templates as they are built and pre-deployment.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/integrations/run-tasks/index.mdx:150]:
```suggestion
[Rapid7's](https://rapid7.com/) InsightCloudSec enables security teams to provide a single, consistent definition of good across the full CI/CD pipeline, allowing them to evaluate potential risks of Infrastructure as Code (IaC) templates during building and pre-deployment.
```

Suggest rewording "as they are built and pre-deployment" to "during building and pre-deployment" to avoid passive voice and maintain phrasing consistency.

**Review by mgarrell777 — APPROVED** [review]
@gautambaghel Thanks for your work on this PR. A small suggested change, but looks good to me overall.

**Comment by gautambaghel** [comment]
@mgarrell777 — ready to merge
